policy/scan/local_scanner.go

```diff
@@ -255,33 +253,34 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstreamConf
 			select {
 			case <-ctx.Done():
 				log.Warn().Msg("request context has been canceled")
+				// When we scan concurrently, we need to call Errored(asset.Mrn) status for this asset
+				multiprogress.Close()
```

Comment:
i might have missed this in the cnquery review, but is it ok for close to get called for each asset?

Reply:
As I understood this part, currently, we cancel the complete scan. But yes, we need to change this as soon as we migrate to concurrent scans.